public/js/ui.js

```diff
@@ -120,6 +120,30 @@ if (!linz) {
 
         });
 
+        // add ability to open URL in a modal
+        $('[data-linz-modal]').click(function () {
+
+            if ($(this).attr('data-linz-disabled') === 'true') {
+                return;
+            }
+
+            var queryObj = $(this),
+                url = queryObj[0].nodeName === 'BUTTON' ? queryObj.attr('data-href') : queryObj.attr('href');
+
+            $('#linzModal').modal().load(url, function () {});
```

Comment:
Seeing as the `load` function is provided by jQuery, let's support the entire api which is `.load( url [, data ] [, complete ] )`. As per the `load` function, `data` and `complete` should be optional.